tests/Auth/AuthPasswordBrokerTest.php

```diff
@@ -53,13 +64,11 @@ public function testBrokerCreatesTokenAndRedirectsWithoutError()
         $mocks = $this->getMocks();
         $broker = $this->getMockBuilder(PasswordBroker::class)->setMethods(['emailResetLink', 'getUri'])->setConstructorArgs(array_values($mocks))->getMock();
         $mocks['users']->shouldReceive('retrieveByCredentials')->once()->with(['foo'])->andReturn($user = m::mock(CanResetPassword::class));
+        $mocks['tokens']->shouldReceive('recentlyCreated')->once()->with($user)->andReturn(false);
```

Comment:
I would rather leave the original mock for all the test where the new one is not required to test the original functionality too. This also helps keeping existing tests unchanged.